### PR TITLE
Added my(194h) atreus62 keymap

### PR DESCRIPTION
**Pull request by 194h**

### CODE REVIEW

File: keyboards/atreus62/keymaps/194h/config.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#define ONESHOT_TIMEOUT 3000
+#define TAPPING_TERM 200
+#define PREVENT_STUCK_MODIFIERS
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/config.h]: This line can be safely deleted, as this behavior is now the default for QMK.

File: keyboards/atreus62/keymaps/194h/config.h [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+#define FORCE_NKRO
+#define LEADER_TIMEOUT 1000
+#define TAPPING_TOGGLE 3
+
+#include "../../config.h"
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/config.h]: This line can be safely deleted as the build script includes all `config.h` files in the directory tree automatically.

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 3]
```diff
@@ -0,0 +1,182 @@
+#include "atreus62.h"
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/keymap.c]: Lines 1-3 should be changed to `#include QMK_KEYBOARD_H`, which includes all of these files automatically.

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+//Layers
+#define L1 0
+#define L2 1
+#define L3 2
+#define L4 3
+#define L5 4
+
+// Fillers to make layering more clear
+#define _____ KC_TRNS
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/keymap.c]: For Lines 69-70, QMK defines keycodes to do this automatically.

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L29-L31

Please delete these lines, and replace all instances of  
`_____` (five underscores) with
`_______` (seven underscores), and all instances of  
`XXXXX` (five Xs) with  
`XXXXXXX` (seven Xs).

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+
+// Momentary switch to layer - One Shot Layer
+#define OSL_L3 OSL(L3)
+#define OSL_L4 OSL(L4)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [L1] = {
+
+	//  commented_row  _____________  _____________  _____________  _____________  _____________  to_even_out__  _____________  _____________  _____________  _____________  _____________  the_columns__   //
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/keymap.c]: All your layers should be formatted to use a layout macro that resembles the physical layout of the keyboard, like this:

https://github.com/qmk/qmk_firmware/blob/4a7baaaf50bc9cce3886ff7e1b6455e028d92ab9/keyboards/atreus62/keymaps/default/keymap.c#L15-L21

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+	{   XXXXX          ,TO(L1)        ,TO(L2)        ,TO(L3)        ,TO(L4)        ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         },
+	{   XXXXX          ,XXXXX         ,XXXXX         ,XXXXX         ,RESET         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,NO_AO         },
+	{   XXXXX          ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,NO_OE         ,NO_AE         },
+	{   XXXXX          ,XXXXX         ,XXXXX         ,KC_CAPS       ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         },
+	{   XXXXX          ,_____         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         ,XXXXX         }
+
+  }
+};
+const uint16_t PROGMEM fn_actions[] = { };
```
Comment (noroadsleft) [inline keyboards/atreus62/keymaps/194h/keymap.c]: This line can be safely deleted as there's no functionality here, and the function itself is deprecated.

**Comment by noroadsleft** [comment]
Here's another suggestion, but it doesn't pertain to your submitted files.

I noticed that you've filed this PR from your fork's `master` branch. While this isn't a major issue now, if you change your keymap later, you'll run into merge conflicts if you try to file another PR because the commit history will have diverged between your master and QMK's.

It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing. I keep my keymap in a branch named `keymaps/noroadsleft`.

Usually a couple times a day, I do the following (using MSYS2 on Windows 8):

```
git fetch upstream
git checkout master
git pull upstream master
git push origin master
```

This retrieves any recent changes to QMK's master - nicknamed `upstream` - and syncs them to my fork, which is nicknamed `origin`. Then when I start development on something, I create a new branch for whatever I'm working on with `git checkout -b dev_branch`, which creates a branch `dev_branch` that is based on the current branch, and then checks out the new branch. I'll make all my changes to `dev_branch`, then when they're ready, I'll do a pull request from `noroadsleft:dev_branch` to `qmk:master`.

If you need any help with this just reply. In particular, the `upstream` alias requires some additional setup to work.

If you are using a different shell or a GUI client, this is still possible.

**Comment by 194h** [comment]
Thanks for the help.
I'll try to incorporate cleaner development practices as well going forward, working in a branch, as you suggested.

**Comment by drashna** [comment]
Awesome, looks good!

**Pull request merged**